deploy/docker-compose/README

```diff
@@ -0,0 +1,153 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
```

Comment:
NIT: "minimal"? no, not in terms of functionality. We call it Lite version, maybe "light-weight" is better here, and we can explain "Lite" means light on technology stack dependencies and and user familiarity with newer technologies.

Reply:
@dbericat ?

Reply:
Added my comments to the review. Copying here as well:

"As described on the MONAI Operating Environments guideline, the journey from development to production usually requires multiple steps across different environments, operated by different teams and with different requirements.

MONAI Deploy Lite is designed to facilitate the testing and validation of MAPs in the early stages of this pipeline (i.e. workstation environment), where ease of use and time to get started is most important."

Reply:
Updated